flowblocks/hero.ini

```diff
@@ -0,0 +1,2 @@
+[block]
+name = Hero
```

Comment:
I ask this from my ignorance but do you know if there's a chance to use the text from the as variables defined here? maybe it could be easier to handle in the future

Reply:
About that, I am trying to explore jinja templates. And I felt this section particularly has very less content so I thought that can be handled by the template. But I will update it because obviously it will increase readability